lib/companies/slack.ex

```diff
@@ -2,7 +2,23 @@ defmodule Companies.Slack do
   use HTTPoison.Base
 
   def notify(%{action: action, resource: resource, user: %{nickname: nickname}}) do
-    message = %{
+    case notify_slack() do
```

Comment:
I think we should configure which module to use rather than this bool. Here we lose our notifications in dev which is not useful.

```elixir
defmodule Notify.Console do
  use HTTPoison.Base

  require Logger

  def notify(type, _resource), do: Logger.info("NOTIFICATION — #{type}")
end
```